### PR TITLE
Scraping Data from Flipkart Website using Python

### PR DESCRIPTION
**Pull request by ShivamKumarSah**
In this program, I have scraped the mobile phone prices, ratings, and model name from Flipkart, which is one of the popular e-commerce websites.

# Description

A short summary of what is included in your Pull Request.

Fixes #(issue_no)

Replace `issue_no` in the above line, with the issue related to this PR.

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [x] I have named my files and folder, according to this project's guidelines.
- [x] My code follows the style guidelines of this project.
- [ ] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [ ] I have included a requirements.txt file (if external libraries are required.)
- [x] My changes do not produce any warnings.
- [x] I have added a working sample/screenshot of the script.

### CODE REVIEW

File: Web_scraping_using_python [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+page_soup = soup(page_html, features="html.parser")  
+  
+# print(soup.prettify(containers[0]))  
+  
+# This variable held all html of webpage  
+containers = page_soup.find_all("div",{"class": "_3O0U0u"})  
+# container = containers[0]  
+# # print(soup.prettify(container))  
+#  
+# price = container.find_all("div",{"class": "col col-5-12 _2o7WAb"})
```
Comment (chavarera) [inline Web_scraping_using_python:21]: do not put any commented code please remove it
```suggestion
# price = container.find_all("div",{"class": "col col-5-12 _2o7WAb"})  
```

File: Web_scraping_using_python [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  
+# Request from the webpage  
+myurl = "https://www.flipkart.com/search?q=iphones&otracker=search&otracker1=search&marketplace=FLIPKART&as-show=on&as=off"  
+  
+  
+uClient  = uReq(myurl)  
+page_html = uClient.read()  
+uClient.close()  
+  
+page_soup = soup(page_html, features="html.parser")
```
Comment (chavarera) [inline Web_scraping_using_python:12]: create a function to get beautiful soup object

**Review by chavarera — CHANGES_REQUESTED** [review]
Please add proper Readme.md

**Review by AdityaJ7 — CHANGES_REQUESTED** [review]
Make the changes as requested by chavarera

**Comment by tusharnankani** [comment]
Are you working on it? @ShivamKumarSah

**Review by chavarera — CHANGES_REQUESTED** [review]
@ShivamKumarSah are you working on it?

**Pull request closed without merging**